src/main/kotlin/com/openlattice/chronicle/services/studies/StudyService.kt

```diff
@@ -136,4 +182,22 @@ class StudyService(
 
             }
     }
+
+    override fun updateStudy(connection: Connection, studyId: UUID, study: StudyUpdate) {
+        val ps = connection.prepareStatement(UPDATE_STUDY_SQL)
+
+        ps.setObject(1, study.title)
+        ps.setObject(2, study.description)
+        ps.setObject(3, OffsetDateTime.now())
+        ps.setObject(4, study.startedAt)
+        ps.setObject(5, study.endedAt)
+        ps.setObject(6, study.lat)
+        ps.setObject(7, study.lon)
+        ps.setObject(8, study.group)
+        ps.setObject(9, study.version)
+        ps.setObject(10, study.contact)
+        ps.setString(11, mapper.writeValueAsString(study.settings))
+        ps.setObject(12, studyId)
+        ps.executeUpdate()
```

Comment:
Not every field should be set with `setObject`. For example, `title` and `description` should be `setString`. Also, I'm pretty sure we've had bugs where the index has been wrong over time when it's managed manually. Using a variable will prevent that:

```
var index = 1
ps.setString(index++, study.title)
ps.setString(index++, study.description)
...
ps.setObject(index, studyId)
```